packages/ag-charts-enterprise/src/series/waterfall/waterfallThemes.ts

```diff
@@ -1,27 +1,25 @@
 import { _Theme } from 'ag-charts-community';
 
+const itemTheme = {
+    strokeWidth: 0,
+    label: {
+        fontStyle: undefined,
+        fontWeight: _Theme.FONT_WEIGHT.NORMAL,
+        fontSize: 12,
+        fontFamily: _Theme.DEFAULT_FONT_FAMILY,
+        color: _Theme.DEFAULT_LABEL_COLOUR,
```

Comment:
This colour wasn't being set by default, so was using the default fill (which I removed)